addin-proximity/dist/proximity.html

```diff
@@ -1 +1,77 @@
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Proximity</title>
+    <link rel="stylesheet"
+        href="https://cdn.jsdelivr.net/gh/Geotab/sdk-addin-samples@master/addin-proximity/dist/styles/vendor.css?1693340755">
+    <link rel="stylesheet"
+        href="https://cdn.jsdelivr.net/gh/Geotab/sdk-addin-samples@master/addin-proximity/dist/styles/main.css?1693340755">
+    <script
+        src="https://cdn.jsdelivr.net/gh/Geotab/sdk-addin-samples@master/addin-proximity/dist/scripts/vendor.js?1693340755"></script>
+    <script
+        src="https://cdn.jsdelivr.net/gh/Geotab/sdk-addin-samples@master/addin-proximity/dist/scripts/main.js?1693340755"></script>
+</head>
+
+<body>
+    <div id="proximity" style="height: 100%">
+        <div id="disclaimer"><label id="disclaimerLabel">Sample add-in/tool</label>
+            <p>This tool is provided as an example and is available on an As-Is basis. You must assume all the risks and
+                costs associated with the use of the sample tool, including, without limitation, any damage to any
+                equipment, software, information or data. In addition, we are not obligated to provide any maintenance,
+                technical or other support for the sample tool. In order to improve our products, we may at any time,
+                and without warning, change the design, presentation, or functionality of the software.</p>
+        </div>
+        <div class="intgr-notification intgr-notification--info">
```

Comment:
The integration style sheet isn't used in the samples, so let's just use inline styling for the warning
```suggestion
        <div style="background-color: #FFF2D2; border: 1px solid #FFF2D2; color: #784600; overflow: hidden; padding: 8px 35px 8px 14px; border-radius: 4px; position: relative;">
```